processing/src/main/java/io/druid/segment/IndexIO.java

```diff
@@ -748,7 +808,7 @@ public int size()
             channel.write(ByteBuffer.wrap(specBytes));
             serdeficator.write(channel);
             channel.close();
-          } else if (filename.startsWith("met_")) {
+          } else if (filename.startsWith("met_") || filename.startsWith("numeric_dim_")) {
```

Comment:
Ok cool, could you slap a quick comment in the code that the approach is deprecated not intended to be used to add more dimension types, instead favoring the v9 approach or later?